pkgs/tools/security/cve-bin-tool/default.nix

```diff
@@ -65,7 +67,11 @@ buildPythonApplication rec {
     cchardet
     # needed by brotlipy
     pillow
-  ];
+  ] ++ (with pythonPackages; [
```

Comment:
pythonPackages shouldn't actually be necessary, just adding setuptools to the inputs will work

Reply:
thanks! I thought I tried without pythonPackages, I may have done wrong first.